plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -94,6 +96,11 @@ func (p *Plugin) ValidateInitialization() error {
 	if p.podsGetter == nil {
 		return fmt.Errorf("%s requires a pod getter", PluginName)
 	}
+	if p.features.Enabled(features.MirrorPodNodeRestriction) {
```

Comment:
will need to react once https://github.com/kubernetes/kubernetes/pull/84813/files#diff-cbf05e22a4f412de387669881892d46b merges